packages/yamjs-test-util/src/server/setup/download.ts

```diff
@@ -1,21 +1,21 @@
 export const downloadPaper = async (
-  url = 'https://api.papermc.io/v2/projects/paper/versions/1.19.3/builds/448/downloads/paper-1.19.3-448.jar'
+   url = 'https://api.papermc.io/v2/projects/paper/versions/1.20.1/builds/177/downloads/paper-1.20.1-177.jar'
 ) => {
-  const response = await fetch(url, { method: 'GET' })
+   const response = await fetch(url, { method: 'GET' });
 
-  if (!response) throw new Error(`Unable to get ${url}`)
+   if (!response) throw new Error(`Unable to get ${url}`);
 
-  return Buffer.from(await response.arrayBuffer())
-}
+   return Buffer.from(await response.arrayBuffer());
+};
 
 export const downloadYamJs = async (version: string = '0.0.1', isLegacy = false) => {
-  const response = await fetch(
-    `https://github.com/Yam-JS/YamJS/releases/download/${version}/yamjs-paper-${
-      isLegacy ? 'legacy-' : ''
-    }${version}.jar`
-  )
+   const response = await fetch(
+      `https://github.com/Yam-JS/YamJS/releases/download/${version}/yamjs-paper-${
+         isLegacy ? 'legacy-' : ''
+      }${version}.jar`
+   );
 
-  if (!response) throw new Error(`Unable to get YamJS plugin`)
+   if (!response) throw new Error(`Unable to get YamJS plugin`);
 
-  return Buffer.from(await response.arrayBuffer())
-}
+   return Buffer.from(await response.arrayBuffer());
+};
```

Comment:
You seem to be re-formatting the file. Can we undo this or use  prettier formatting?

Reply:
Funny you say that, this is what Prettier formatted to. With that said, the useless spaces are annoying. Should be fixed.